src/components/Cards/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,120 @@
+import { Link } from "react-router-dom";
+import { bxIcons } from "../../../data/icons";
+import "./VideoCard.css";
+
+export const VideoCard = ({ video }) => {
+	const {
+		_id,
+		contentDetails: { duration },
+		snippet: {
+			categoryId,
+			channelId,
+			channelTitle,
+			description,
+			localized: { title },
+			publishedAt,
+			tags,
+			thumbnails: {
+				high: { url: thumbnailURL },
+			},
+		},
+		statistics: { viewCount },
+	} = video;
+
+	console.log({
+		_id,
+		title,
+		description,
+		categoryId,
+		channelId,
+		channelTitle,
+		publishedAt,
+		tags,
+		thumbnailURL,
+		viewCount,
+	});
+
+	const getIndianDate = (isoDate) => {
+		let dateObj = new Date(isoDate);
+		return dateObj.toLocaleDateString("en-IN");
+	};
+
+	const getDuration = (str) => {
```

Comment:
change from "getDuration" to "getVideoDuration" it will be more understandable

Reply:
will do that